custom/ucla/tests.py

```diff
@@ -0,0 +1,141 @@
+            domain=self.domain,
+            data_type_id=self.data_type.get_id,
+            fields={
+                "risk_profile": FieldList(
+                    field_list=[
+                        FixtureItemField(
+                            field_value=risk,
+                            properties={}
+                        )
+                    ]
+                ),
+                "sequence": FieldList(
+                    field_list=[
+                        FixtureItemField(
+                            field_value=sequence,
+                            properties={}
+                        )
+                    ]
+                ),
+                "message": FieldList(
+                    field_list=[
+                        FixtureItemField(
+                            field_value=message,
+                            properties={}
+                        )
+                    ]
+                ),
+            },
+            item_attributes={},
+        )
+        data_item.save()
+        self.addCleanup(data_item.delete)
+
+    @run_with_all_backends
+    def test_message_bank_doesnt_exist(self):
+        with create_test_case(self.domain, self.case_type, 'test-case') as case:
+            self.assertIsNone(ucla_message_bank_content(self._reminder(), self._handler(), case))
+
+    @run_with_all_backends
+    def test_message_bank_doesnt_have_correct_properties(self):
+        data_type = FixtureDataType(
+            domain=self.domain,
+            tag=self.fixture_name,
+            fields=[],
+            item_attributes=[]
+        )
+        data_type.save()
+        self.addCleanup(data_type.delete)
+        with create_test_case(self.domain, self.case_type, 'test-case') as case:
+            self.assertIsNone(ucla_message_bank_content(self._reminder(), self._handler(), case))
+
+    def test_not_passing_case(self):
+        self._setup_fixture_type()
+        self.assertIsNone(ucla_message_bank_content(self._reminder(), self._handler(), None))
+
+    @run_with_all_backends
+    def test_passing_case_without_risk_profile(self):
+        self._setup_fixture_type()
+        with create_test_case(self.domain, self.case_type, 'test-case') as case:
+            self.assertIsNone(ucla_message_bank_content(self._reminder(), self._handler(), case))
+
+    @run_with_all_backends
+    def test_no_relevant_message_invalid_risk(self):
+        self._setup_fixture_type()
+        with create_test_case(self.domain, self.case_type, 'test-case', {'risk_profile': 'risk2'}) as case:
+            self.assertIsNone(ucla_message_bank_content(self._reminder(), self._handler(), case))
+
+    @run_with_all_backends
+    def test_no_relevant_message_invalid_seq_num(self):
+        self._setup_fixture_type()
+        with create_test_case(self.domain, self.case_type, 'test-case', {'risk_profile': 'risk1'}) as case:
+            self.assertIsNone(ucla_message_bank_content(self._reminder(), self._handler(num_events=1), case))
```

Comment:
we could use a reminder with schedule_iteration_num=2 for this test, keeping num_events=1 on the handler